app/controllers/v5/stories_controller.rb

```diff
@@ -0,0 +1,93 @@
+module V5
+  class StoriesController < V5::BaseUserController
+    power :stories, map: {
+      [:show] => :show_stories,
+      [:update] => :update_stories,
+      [:create] => :create_story
+    }, as: :story_scope
+
+    def index
+      load_stories
+      render_stories
+    end
+
+    def show
+      load_story or render_not_found
+      render_story
+    end
+
+    def create
+      if build_story
+        render_story :created
+      else
+        render_errors
+      end
+    end
+
+    def update
+      load_story or render_error('Invalid story id') && return
```

Comment:
load_story or render_error('Invalid story id') and return -- (fails)
render_error('Invalid story id') and return unless load_story -- (passes)
return render_error('Invalid story id') unless load_story -- (passes)
Which looks/feels correct?